src/components/app/components/AppLink.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const AppLink = ({ to, children }) => <NavLink to={to}>{children}</NavLink>;
```

Comment:
what is happening here

Reply:
this is javascript destructing of the argument. It is the same as:
```
const AppLink = prop => {
    const { to, children } = prop;
    return <NavLink to={to}>{children}</NavLink>;
}
```

Reply:
ok